### PR TITLE
refactor: migrate from `slice.items` to `slice.primary` groups

### PR DESCRIPTION
**Pull request by lihbr**
Contributes to: [DT-2080](https://linear.app/prismic/issue/DT-2080/aadevuser-i-can-see-the-nuxt-starters-updated-with-the-slice-model)

### Context

This PR migrates slices from using `slice.items` array to using regular group fields within the `slice.primary` section.

### Before Merging

- [x] Update content dump (WIP Michel)
- [x] Update repositories on production (once backend is on production)
- [x] Revert `slicemachine.config.json`

### CODE REVIEW

File: slices/Hero/index.vue [hunk 1]
```diff
@@ -40,14 +40,14 @@ defineProps(getSliceComponentProps<Content.HeroSlice>(
         class="flex flex-wrap gap-4"
```
Comment (levimykel) [inline slices/Hero/index.vue:40]: The line above this needs to be updated with `v-if="slice.primary.buttons.length"`

**Review by levimykel — CHANGES_REQUESTED** [review]
One small change is requested. I approve the PR once this is fixed.

File: slices/Hero/index.vue (hunk 1 above)
Comment (lihbr) [reply slices/Hero/index.vue:40]: Oh, nice catch, thanks!

**Review by levimykel — APPROVED** [review]
Looks good, thanks.

**Pull request merged**